tests/api-watch/test_diff_modtpe_decl.ml

```diff
@@ -0,0 +1,75 @@
+open Api_watch
+open Test_helpers
+
+let%expect_test "Module_type with value changes" =
+  let reference =
+    compile_interface
+      {|
+    module type M = sig
+      val g : int -> string
+    end
+  |}
+  in
+  let current = compile_interface {|
+    module type M 
+  |} in
+  let result = Diff.interface ~module_name:"Main" ~reference ~current in
+  Format.printf "%a" pp_diff_option result;
+  [%expect
+    {| Some (Module Main: {Modified (Supported [ Module_type M: {Modified (Unsupported)}])}) |}]
+
+let%expect_test "Module_type with module change" =
+  let reference =
+    compile_interface
+      {|
+    module type M = sig
+      val b : int list -> int
+    end
+    module P : sig val x: float end
```

Comment:
You can remove this as it is just noise for the test. It polutes the output with a removed module which is already tested elsewhere.